website/contributing/release-troubleshooting.md

```diff
@@ -3,6 +3,53 @@ id: release-troubleshooting
 title: Troubleshooting
 ---
 
+### Adding a Local Commit in a -stable branch
+
+In the `-stable` branches (contrary to main branch), the release crew can directly push new commits to quickly address issues. After applying the necessary changes, those changes need to be re-implemented in `main`; this is to avoid inconsistent state across branches, and to prevent having the same failures in the future releases.
```

Comment:
```suggestion
In the `-stable` branches (contrary to main branch), the release crew can directly push new commits to quickly address issues. After applying the necessary changes, those changes need to be re-implemented in `main`; this is to avoid inconsistent state across branches, and to prevent having the same failures in future releases.
```